conf/messages.en

```diff
@@ -72,6 +72,12 @@ document.packageType.checkYourAnswersLabel = Package type
 document.packageType.change.hidden = package type
 document.packageType.error.required = Select the type of package the goods arrived in
 
+document.addLineItemNumberYesNo.title = Do you want to add a line item number?
+document.addLineItemNumberYesNo.heading = Do you want to add a line item number?
+document.addLineItemNumberYesNo.checkYourAnswersLabel = Line item number
```

Comment:
Hello Kev, I believe the value for the checkYourAnswerLabel here is more suited to the actual lineItemNumber view itself. This could probably retain the same value as the title.

Reply:
I took the values from the CYA page content doc (though there is a possibility they could be removed altogether)